packages/commons/src/config/readmodelDbConfig.ts

```diff
@@ -17,7 +17,3 @@ export const ReadModelDbConfig = z
   }));
 
 export type ReadModelDbConfig = z.infer<typeof ReadModelDbConfig>;
-
-export const readmodelDbConfig: ReadModelDbConfig = ReadModelDbConfig.parse(
```

Comment:
Unused, and I am removing it for the same reason described above (parsing on startup would require adding these env vars to make the common tests run work)